theme/templates/accounts/profile.html

```diff
@@ -86,7 +86,6 @@
 
                                     {#        Account deactivation       #}
                                     <div><a data-toggle="modal" data-target="#deactivate-account-dialog">Deactivate HydroShare account</a></div>
-                                    <div><a href="/accounts/update">Change HydroShare API password</a></div>
```

Comment:
This is the only mention of the API password that I am aware of outside of the help documentation.

Reply:
believe that's right